packages/preview-middleware-client/src/flp/enableFakeConnector.ts

```diff
@@ -0,0 +1,120 @@
+import LrepConnector from 'sap/ui/fl/LrepConnector';
+import FakeLrepConnector from 'sap/ui/fl/FakeLrepConnector';
+import type { FlexSettings } from 'sap/ui/rta/RuntimeAuthoring';
+
+interface FlexChange {
+    [key: string]: string | object | undefined;
+    support: {
+        generator?: string;
+    };
+}
```

Comment:
Can we use this type in both connectors?

Reply:
Changed in: [a905279](https://github.com/SAP/open-ux-tools/pull/1597/commits/a9052798597352c89f85d4980d523bc49965070e)